.schema/config.schema.json

```diff
@@ -972,7 +987,8 @@
   "required": [
     "identity",
     "dsn",
-    "selfservice"
+    "selfservice",
+    "version"
```

Comment:
Does `version` really have to be required?

Reply:
right we wanted to print a warning in case the version is not specified or does not match